.github/workflows/build-push.yml

```diff
@@ -54,9 +54,12 @@ jobs:
     strategy:
       fail-fast: ${{ github.event_name == 'pull_request' }}
       matrix:
-        container_flavor:
-          - master
-          - worker
+        include:
+          - container_flavor: master
+            container_verify_string: "buildmaster configured in /master"
```

Comment:
Wouldn't this approach get hairy once we would like to add more checks/tests?

Reply:
Mhhh yes...

I wonder if the test will be done all on logs tho...

If we will check that then we might consider adding all the check on a shell script and just run it here.

But we you are right if the intention is to add additional stuff then I think the current way of keep the master and worker split is the only way...

MAYBE I can split the get log part generalize the command to run the container?

Reply:
Well its actually very lame testing, but better then no testing.

In the long term I planned to look into labgrid's [DockerStrategy](https://labgrid.readthedocs.io/en/latest/configuration.html#dockerstrategy) and use that for more advanced QA, the similar approach could be then re-used for SDK/ImageBuilder/rootfs container testing in the openwrt/docker repository etc.

Reply:
```
buildbot.errors.PluginDBError: Requirements are not satisfied for buildbot.www:base: The 'zope-interface>=5' distribution was not found and is required by Twisted
```
Murphy's law? :P https://github.com/openwrt/buildbot/actions/runs/6867270540/job/18678271015#step:5:51

Reply:
Yes I notice the same while testing this pr. Thing it's there from a lot :(

Reply:
About Murphy's  law... I had to rerun the test as for some reason the github instance lost connection ahahaha